spec/integration/resource/api300/c7000/login_detail/create.rb

```diff
@@ -0,0 +1,18 @@
+# (C) Copyright 2017 Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+require 'spec_helper'
+
+klass = OneviewSDK::API300::C7000::LoginDetail
+RSpec.describe klass, integration: true, type: CREATE, sequence: seq(klass) do
+  let(:current_client) { $client_300 }
+  include_examples 'LoginDetailExample', 'integration api300 context'
```

Comment:
The `include_examples` is including the 'LoginDetailExample', but the name of shared example created was ['LoginDetailGetExample'](https://github.com/HewlettPackard/oneview-sdk-ruby/pull/297/commits/f71dbb565615b657ffffc84e2fe1ed4f88d27a5d#diff-06409f5bbce4ed4cde67a23b4600f8b0R12).